docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,734 @@
+            operator = X ^ X
+
+     -
+
+        .. code-block:: python
+
+            from qiskit.quantum_info import Pauli
+            X = Pauli('X')
+            op = X ^ X
+
+     -
+
+        .. code-block:: python
+
+            from qiskit.quantum_info import Pauli, SparsePauliOp
+            op = Pauli('X') ^ Pauli('X')
+
+            # equivalent to:
+            op = SparsePauliOp('XX')
+
+Common non-parametrized gates (Clifford)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. list-table:: Migration of ``qiskit.opflow.operator_globals`` (2/3)
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+
+   * - :class:`~qiskit.opflow.CX`, :class:`~qiskit.opflow.S`, :class:`~qiskit.opflow.H`, :class:`~qiskit.opflow.T`, :class:`~qiskit.opflow.CZ`, :class:`~qiskit.opflow.Swap`
+     - Append corresponding gate to :class:`~qiskit.QuantumCircuit` + :class:`~qiskit.quantum_info.Clifford` + ``.to_operator()``
+     - This alternative is not too efficient, as ``.to_operator()`` is dense and scales exponentially with the size of the circuit.
+
+   * -
+
+        .. code-block:: python
+
+            from qiskit.opflow import H
+            op = H ^ H
+
+     -
+
+        .. code-block:: python
+
+            from qiskit import QuantumCircuit
+            from qiskit.quantum_info import Clifford
+            qc = QuantumCircuit(2)
+            qc.h(0)
+            qc.h(1)
+            op = Clifford(qc).to_operator()
+
+            # or...
+            qc = QuantumCircuit(1)
+            qc.h(0)
+            H = Clifford(qc).to_operator()
+            op = H ^ H
+
+     -
+
+1-Qubit States
+~~~~~~~~~~~~~~
+.. list-table:: Migration of ``qiskit.opflow.operator_globals`` (3/3)
+   :header-rows: 1
+
+   * - opflow
+     - alternative
+     - notes
+
+   * - :class:`~qiskit.opflow.Zero`, :class:`~qiskit.opflow.One`, :class:`~qiskit.opflow.Plus`, :class:`~qiskit.opflow.Minus`
+     - :class:`~qiskit.quantum_info.StabilizerState` or :class:`~qiskit.quantum_info.Statevector` or :class:`~qiskit.QuantumCircuit`, depending on the use case
+     - In principle, :class:`~qiskit.quantum_info.StabilizerState` is the most efficient replacement for :class:`~qiskit.opflow` states, but the functionality is not identical. See API ref. for more info.
+
+   * -
+
+        .. code-block:: python
+
+            from qiskit.opflow import Zero, One, Plus, Minus
+
+            state1 = Zero ^ One
+            state2 = Plus ^ Minus
+
+     -
+
+        .. code-block:: python
+
+            from qiskit import QuantumCircuit
+            from qiskit.quantum_info import StabilizerState
+
+            qc_zero = QuantumCircuit(1)
+            qc_one = copy(qc_zero)
+            qc_one.x(0)
+            state1 = StabilizerState(qc_zero) ^ StabilizerState(qc_one)
+
+            qc_plus = copy(qc_zero)
+            qc_plus.h(0)
+            qc_minus = copy(qc_one)
+            qc_minus.h(0)
+            state2 = StabilizerState(qc_plus) ^ StabilizerState(qc_minus)
```

Comment:
I know, but I don't know what better format we could use here. It looks like the tables don't adapt to the screen size too, if they did, these examples would probably not wrap.

Reply:
I have decided to merge the "alternative" and ''notes" columns, as the third one was almost empty. This has improved a bit the look of the tables. But still, I think there are probably other ways in which this could be improved.

Reply:
In other guides we have more done things sequentially as below. Here one gets a full width for the code. The side-by-side for more compact examples seems nicer, but if the code starts wrapping it rather detracts from that for me.


**Previously**
```
old way example code
```

**New**
```
new way example code
```